docs/experience_date_search/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Experience date search
+has_children: true
+nav_order: 8
+has_toc: false
+---
+
+# Experience date search
+
+Experience date search enables you to filter a Creator's experiences with a separate result for each departure date of each experience.
+
+The experience date search is executed using the [experience_date_search tag]({% link docs/reference/tags/experience_date_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object and a `search` [Search]({% link docs/reference/objects/search_query.md %}) object.
+
+The experience date search will only return public, published dates i.e. where the experience is published and has not been marked as private under 'Manage product availability' in the product settings and where the date is not hidden TBC UI on this.
```

Comment:
> and where the date is not hidden TBC UI on this.

Happy to leave this in if we have a ticket to update the docs when we add 'hide' feature, or maybe just take this out.

Reply:
Good catch. I'll remove the TBC actually,